src/server.c

```diff
@@ -3419,28 +3285,26 @@ static void propagatePendingCommands(void) {
     /* In case a command that may modify random keys was run *directly*
      * (i.e. not from within a script, MULTI/EXEC, RM_Call, etc.) we want
      * to avoid using a transaction (much like active-expire) */
-    if (server.current_client &&
-        server.current_client->cmd &&
-        server.current_client->cmd->flags & CMD_TOUCHES_ARBITRARY_KEYS)
-    {
+    if (server.current_client && server.current_client->cmd &&
+        server.current_client->cmd->flags & CMD_TOUCHES_ARBITRARY_KEYS) {
         transaction = 0;
     }
```

Comment:
What happened to the attach multiline rule?

In general, I think it's much better now and total changes lines are much less: 30k instead of 40k. 

Btw, you didn't commit the clang format config file?

Reply:
added .clang-format.

>What happened to the attach multiline rule?

it is attached, right?

Reply:
Yes, but I thought we wanted to attach single line but detach when multiline, BraceWrappingAfterControlStatementStyle MultiLine. That was the old convention anyway and I think most of the code followed this style, but I may have missed some discussion here and it's no big deal for me. I just noticed the change.

Reply:
Got it. We will always attach.

Reply:
Ok

Reply:
I was noticing the number of multi-line control statements has gone down significantly, so attaching the braces probably is just a lot less important.

Reply:
Yes, anyway, my convern was always that there's no indentation difference between secondlongline and thirdlongline in the example below, but now I've come to the conclusion that this is never really a problem. The important thing is that it's easy to see where the whole `if` starts and ends (`}`).

```c
if (alonglongline ||
    secondlongline) {   // <-- 
    thirdlongline;   // <-- same indentation
    morelines;
}

Reply:
Oh, that's strange.  Most formatters that I have seen indent `secondlongline` one more time.